setup-homebrew/main.sh

```diff
@@ -83,8 +83,8 @@ fi
 GEMS_PATH="$HOMEBREW_REPOSITORY/Library/Homebrew/vendor/bundle/ruby/"
 GEMS_HASH="$(shasum -a 256 "$HOMEBREW_REPOSITORY/Library/Homebrew/Gemfile.lock" | cut -f1 -d' ')"
 
-echo "::set-output name=gems-path::$GEMS_PATH"
-echo "::set-output name=gems-hash::$GEMS_HASH"
+echo "gems-path=$GEMS_PATH" >> $GITHUB_OUTPUT
```

Comment:
Does this work no issue in our container with non-root user?

Reply:
Yeah this probably needs a chown

Reply:
I wonder if we can do something here that can work globally for subsequent actions.

Maybe here or something: https://github.com/Homebrew/actions/blob/4019aa33c7aaa82f22b10bdf76fe6ef47b406c61/setup-homebrew/main.sh#L51

Reply:
> I wonder if we can do something here that can work globally for subsequent actions.

Good idea 👍🏻